Javid_Alizada/02.GIT.Local/README.md

```diff
@@ -0,0 +1,37 @@
+- mkdir 02.GIT.Local
```

Comment:
what is `-` is doing here? And where is markup?

Reply:
Thanks. Changed dash's to numeric value and added markup please recheck.